README.md

```diff
@@ -170,6 +170,8 @@ When updating configuration values, normally they only become active upon **manu
 
 However, when using the sources `ZooKeeper` or `JCR` it is possible to automatically reinstall the package containing the install hook.
 
+This feature requires AEM 6.3.0 or higher
+
```

Comment:
@kwin  I added the notice in this section instead since it's relevant only for the additional bundle but I can move it to a dedicated section if that's better